docs/testing/debug-with-vs-extension.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Visual Studio Code Debugger Extension for Office Add-ins
+description: 'Use a VS Code extension to debug your Office add-in.'
+ms.date: 05/14/2020
+localization_priority: Normal
+---
+
+# Visual Studio Code Debugger Extension for Office Add-ins
+
+The Visual Studio Debugger Extension for Office Add-ins allows you to use F5 in VS Code to debug your Office Add-in against the Edge runtime. This debugging mode is dynamic, allowing you to set breakpoints while code is running. You can see changes in your code immediately while the debugger is attached, all without losing your debugging session. Your code changes also persist, so you can see the results of multiple changes to your code. The following image shows this extension in action.
+
+![VS Code Debugger Extension debugging a section of Excel Add-ins](../images/vs-debugger-extension-for-office-addins.jpg)
+
+## Pre-Requisites
+
+- [Visual Studio Code](https://code.visualstudio.com/) (must be run as an administrator)
+- [Node.js (version 10+)](https://nodejs.org/)
+- Windows 10
+- [Microsoft Edge](https://www.microsoft.com/edge)
+- [Yo Office](https://github.com/OfficeDev/generator-office)
+
+These instructions assume you have experience using the command line, understand basic JavaScript, and have created an Office add-in project before using the Yo Office generator. If you haven't done this before, consider visiting one of our tutorials, like this [Excel Office Add-in tutorial](../tutorials/excel-tutorial.md).
+
+## Install and use
+
+1. [Create a new project with the latest version of Yo Office](https://docs.microsoft.com/office/dev/add-ins/quickstarts/excel-quickstart-jquery?tabs=yeomangenerator). Follow the prompts within the command line to set up your project. You can choose any language or type of project to suit your needs.
+
+2. Open a command prompt as administrator
+   ![Command prompt options, including "run as administrator" in Windows 10](../images/run-as-administrator-vs-code.jpg)
+
+3. Navigate to your project directory.
+
+4. Run the following command to open your project in Visual Studio Code as an administrator.
+
+```command line
+code .
+```
+
+Once Visual Studio Code is open, navigate manually to the project folder.
+
+> [!TIP]
+> To open Visual Studio Code as an administrator, select the **run as administrator** option when opening Visual Studio Code from the Start Menu in Windows.
+
+5. Within VS Code, select **CTRL + SHIFT + X** to open the Extensions bar. Search for the "VS Code Debugger for Office Add-ins" extension and install it.
```

Comment:
The [MS style guide recommends sentence-style casing](https://styleguides.azurewebsites.net/StyleGuide/Read?id=2700&topicid=27401) for key names and doesn't use spaces.
```suggestion
5. Within VS Code, select **Ctrl+Shift+X** to open the Extensions bar. Search for the "VS Code Debugger for Office Add-ins" extension and install it.
```